services/collaboration/pkg/middleware/tracing.go

```diff
@@ -0,0 +1,40 @@
+package middleware
+
+import (
+	"net/http"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func CollaborationTracingMiddleware(next http.Handler) http.Handler {
```

Comment:
exported func `CollaborationTracingMiddleware` should have comment or be unexported